fjord/suggest/providers/sumo/tests/test_sumosuggest.py

```diff
@@ -13,18 +13,18 @@
     TestCase,
 )
 from fjord.feedback.tests import ResponseFactory
-from fjord.suggest.providers.sumosuggest import (
+from fjord.suggest.providers.sumo import SUMOSuggest
+from fjord.suggest.providers.sumo.provider import (
     SUMO_AAQ_URL,
     SUMO_HOST,
     SUMO_SUGGEST_API_URL,
     SUMO_SUGGEST_SESSION_KEY,
-    SUMOSuggest
 )
 from fjord.suggest.tests import SuggesterTestMixin
 from fjord.redirector.tests import RedirectorTestMixin
 
 
-class SUMOSuggestTestCase(TestCase):
+class SumoTestCase(TestCase):
```

Comment:
My bad. I did a lousy search-and-replace and then instead of undoing it, I went and "fixed things" by hand.

SUMO is an acronym, so it's annoying but I figured SUMO is clearer than Sumo.